app/views/GovukLayoutWrapper.scala.html

```diff
@@ -0,0 +1,132 @@
+@*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers._
+@import uk.gov.hmrc.hmrcfrontend.views.Aliases.UserResearchBanner
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.HmrcLanguageSelect
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.language.{Cy, En, LanguageSelect}
+@import config.AppConfig
+
+@this(
+        config: AppConfig,
+        govukLayout: GovukLayout,
+        govukBackLink: GovukBackLink,
+        hmrcHead: HmrcHead,
+        hmrcStandardHeader: HmrcStandardHeader,
+        hmrcStandardFooter: HmrcStandardFooter,
+        hmrcScripts: HmrcScripts,
+        hmrcReportTechnicalIssueHelper: HmrcReportTechnicalIssueHelper,
+        hmrcLanguageSelect: HmrcLanguageSelect,
+        hmrcTimeoutDialogHelper: HmrcTimeoutDialogHelper
+)
+
+@(
+        pageTitle: String,
+        isUserResearchBannerVisible: Boolean = true,
+        signOut: Boolean = true,
+        timeoutActive: Boolean = true,
+        showBackLink: Boolean = true,
+        empRef: Option[String] = None,
+        showYTALink: Boolean = false
+)(contentBlock: Html)(implicit request: RequestHeader, messages: Messages)
+
+
+@head = {
+    @if(timeoutActive) {
+        @hmrcTimeoutDialogHelper(
+            controllers.routes.HomePageController.signout().url,
+            Some(controllers.routes.SignedOutController.keepAlive().url),
+            Some(config.timedOutUrl),
+            Some(config.timeout),
+            Some(config.timeoutCountdown),
+            None,
+            Some(messages("timeout.message")),
+            keepAliveButtonText = Some(messages("timeout.keepAlive")),
+            signOutButtonText = Some(messages("timeout.signOut"))
+        )
+    }
+
+}
+
+@userResearchBanner = @{
+    if(isUserResearchBannerVisible) {
+        Some(UserResearchBanner(url = config.urBannerLink))
+    } else {
+        None
+    }
+}
+
+@beforeContentBlock = {
+    <div>
+        <div class="hmrc-banner govuk-!-display-inline-block">
+            <div class="hmrc-organisation-logo">
```

Comment:
I think no need to display hmrc logo... could you please check with design team on this?

Reply:
Done the demo to the design team who are happy with the change and still within the current prototype, believe this is outside of scope for this